apmpackage/apm/data_stream/internal_metrics/manifest.yml

```diff
@@ -25,6 +25,8 @@ elasticsearch:
                   type: aggregate_metric_double
                   metrics: [sum, value_count]
                   default_metric: sum
+        event:
+          properties:
```

Comment:
It should be mapped as `doc-value-only` (setting `index: false`).

Reply:
`aggregate_metric_double` does not support `index: false` as mentioned in https://github.com/elastic/apm-server/issues/5243#issuecomment-1351349456

Reply:
ah I missed that, thanks for pointing out.